core/templates/pages/exploration-player-page/services/player-transcript.service.ts

```diff
@@ -43,12 +43,23 @@ export class PlayerTranscriptService {
   // they carry on.
   transcript: StateCard[] = [];
   numAnswersSubmitted = 0;
+
+  private originalTranscript: StateCard[] = [];
```

Comment:
Can you add a clear explanation of the difference between originalTranscript and transcript?

Reply:
Done